src/configupdater/configupdater.py

```diff
@@ -30,8 +30,7 @@
 import re
 import sys
 from abc import ABC
-from collections import OrderedDict as _default_dict
-from collections.abc import MutableMapping
+from collections import OrderedDict
```

Comment:
From Python 3.6 onwards `dict` is ordered by default... is `OrderedDict` still necessary?

Reply:
I would say so as it is more explicit. I read somewhere that no one guarantees that this behaviour will not change in future Python versions.